docs/rfc/rfc-1.adoc

```diff
@@ -267,7 +267,7 @@ base_fee * fee_multiplier`. We repeat until either the transaction posts or
 *Note*: We do not allow users to specify a max btc fee. When users deposit,
 they're agreeing to be swept at whatever fee the operators decide is
 appropriate (based on https://blockstream.info/api/fee-estimates). Operators
-cannot pick a starting fee higher than `base_sweeping_fee_max`.
+cannot pick a starting fee higher than `base_btc_fee_max`.
```

Comment:
Could we mention here that fee cannot be bumped above `btc_fee_max` level?

Reply:
428af5bc8ee9c07f2a4883d66c5351fea66a3525